manifests/config.pp

```diff
@@ -44,8 +44,20 @@
 
   ensure_resource('file', $properties_dir, {'ensure' => 'directory', 'owner' => $user, 'group' => $group} )
 
+  #
+  # Checking if we need to deploy realm file
+  #  ugly, I know. Fix it if you know better way to do that
+  #
   if 'file' in $auth_types {
-
+    $_deploy_realm = true
+  }
+  if 'ldap_shared' in $auth_types {
+    $_deploy_realm = true
+  }
+  if 'active_directory_shared' in $auth_types {
+    $_deploy_realm = true
+  }
```

Comment:
@igalic sorry, I'm afraid I don't understand your question. We need to deploy realm file when "file" or/and "ldap_shared" or/and "active_directory_shared" in use . In case of "shared" approach, we still use realm file for authorization procedure.